selenium/exercise6.rb

```diff
@@ -0,0 +1,21 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path = "C:/Users/JasonGonsalves/Documents/Automation/chromedriver-win64/chromedriver.exe"
+
+driver = Selenium::WebDriver.for :chrome
+
+driver.manage.window.maximize
+
+driver.get "https://letcode.in/forms"
+
+dropdown = driver.find_element(:css, "div:nth-child(5).columns.container > div:last-child.column.is-half > div.field > div.control")
+
```

Comment:
A better approach is to use
`
dropdown = Selenium::WebDriver::Support::Select.new(dropdown_xpath) `

You should make use of the functions mentioned here: https://www.selenium.dev/selenium/docs/api/rb/Selenium/WebDriver/Support/Select.html